src/main/java/uk/gov/digital/ho/hocs/info/api/StandardLineResource.java

```diff
@@ -41,10 +42,13 @@ public StandardLineResource(StandardLineService standardLineService) {
         return ResponseEntity.notFound().build();
     }
 
-    @GetMapping(value = "/topic/{topicUUID}/standardLine", produces = APPLICATION_JSON_UTF8_VALUE)
+    @GetMapping(value = "/topic/{topicUUID}/standardLine", produces = APPLICATION_JSON_VALUE)
```

Comment:
While here, might be an idea just ripping usages of `APPLICATION_JSON_UTF8_VALUE` out...

Reply:
yeah good point.